src/views/List.jsx

```diff
@@ -14,34 +15,47 @@ export function List({ data }) {
 
 	return (
 		<>
-			<label htmlFor="item-search"> Search for an item: </label>
-			<input
-				id="item-search"
-				type="text"
-				placeholder="Search item..."
-				onChange={handleChange}
-				value={searchItem}
-			/>
-			{searchItem && (
-				<button
-					type="button"
-					name="clearInput"
-					aria-label="Clear input"
-					onClick={() => setSearchItem('')}
-				>
-					X
-				</button>
-			)}
-			<ul>
-				{filteredItems.map((item, index) => (
-					<ListItem name={item.name} key={index} />
-				))}
-			</ul>
-
-			{!data.length > 0 && <p>There are no items in this list.</p>}
-
-			{data.length > 0 && !filteredItems.length > 0 && (
-				<p>There are no matching items.</p>
+			{data.length > 0 ? (
```

Comment:
I asked my fellow mentors to give the code a second look since I haven't done this in a bit (😀) and Tim brought up an interesting point! If you wanted to, rather than using a ternary inside a return, you could make this a bit more readable by using an "early return." That would mean checking first if data.length is falsey and returning if so. Sample code:

```
const ExampleComponent = ({data}) => {
  //
  // early return example
  if (!data.length) {
    return <div>...</div>
  }
  return (
    <div> ... </div>
  )
}
```